esphome/components/nabu/audio_reader.cpp

```diff
@@ -80,8 +80,8 @@ esp_err_t AudioReader::start(const std::string &uri, media_player::MediaFileType
   client_config.max_redirection_count = 10;
   client_config.buffer_size = 512;
   client_config.keep_alive_enable = true;
-  client_config.timeout_ms = 250;  // Doesn't raise an error if exceeded in esp-idf v4.4, it just prevents the
-                                   // http_client_read command from blocking for too long
+  client_config.timeout_ms = 5000;  // Doesn't raise an error if exceeded in esp-idf v4.4, it just prevents the
```

Comment:
This should be lower than 5000, if the http server doesn't respond in 5000ms the device will reboot due to the watchdog timeout (5000ms)

Reply:
I believe Kevin has this running in a task. Would a reboot still happen?

Reply:
That's the default setting which we used up until a few days ago. The read command happens inside a task outside of the main loop. Additionally, I believe the http client read blocks when waiting (so other tasks can still run). It shouldn't trigger the watchdog timer.